doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -895,13 +895,26 @@ Modem libraries
 
     * A mention about enabling TF-M logging while using modem traces in the :ref:`modem_trace_module`.
     * The :kconfig:option:`CONFIG_NRF_MODEM_LIB_NET_IF_DOWN_DEFAULT_LTE_DISCONNECT` option, allowing the user to change the behavior of the driver's :c:func:`net_if_down` implementation at build time.
```

Comment:
We would require a changelog entry for libcarrier like here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/releases_and_maturity/releases/release-notes-2.5.0.html#binary-libraries

Reply:
We did not do this for recent releases because our changelog has been linked like this:
![image](https://github.com/nrfconnect/sdk-nrf/assets/38314878/05515a8b-cfc2-454f-975c-1cb34158e2c1)